bash/setup.sh

```diff
@@ -21,6 +21,13 @@ sudo apt-get install openjdk-8-jdk xvfb metacity libvtk6-java libzmq5 libzmq-jav
 # sudo make install
 # cd ..
   
+# -------------------------------------
+# Installing OverTarget
```

Comment:
Just a really small thing: we decided to write Overtarget instead of OverTarget.